### PR TITLE
Time Column

### PR DESCRIPTION
**Pull request by JaySunSyn**
Please merge https://github.com/Neovici/cosmoz-omnitable/pull/38 first

### CODE REVIEW

**Comment by nomego** [comment]
This needs to be rebased, probably make sure of the date behavior introduced with datetime column PR, and maybe evaluated what kind of component to be used for time input. https://github.com/bendavis78/paper-time-picker ?

**Comment by JaySunSyn** [comment]
@nomego please check

File: cosmoz-omnitable-column-time.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+				value="[[ getLocaleString(item, valuePath, userFormat, locale) ]]">
+			</paper-input>
+		</template>
+
+		<template class="header">
+			<paper-dropdown-menu label="[[ title ]]" placeholder="[[ _filterText ]]" title="[[ _tooltip ]]">
+				<div class="dropdown-content" slot="dropdown-content" style="padding: 15px; min-width: 100px;">
+					<h3 style="margin: 0;">[[ title ]]</h3>
+					<paper-input
+						type="time" label="[[ _('From time', t) ]]" step$="[[_filterStep]]" value="{{ _filterInput.start }}">
```
Comment (plequang) [inline cosmoz-omnitable-column-time.html]: I think what you want here is binding to the `step` property, i.e. `step="[[`, not binding to an attribute.

File: cosmoz-omnitable-column-time.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+					</paper-input>
+					<paper-input
+						type="time" label="[[ _('Until time', t) ]]" step$="[[_filterStep]]" value="{{ _filterInput.end }}">
```
Comment (plequang) [inline cosmoz-omnitable-column-time.html]: Again, should use `step="[[`

**Pull request merged**